redhat-appstudio/remote-secret
# Remove migration code

### What does this PR do?
 - Remove migration code
- added extra test to check the situation when data is removed/lost in storage

### Screenshot/screencast of this PR
<img width="1712" alt="Знімок екрана 2023-08-21 о 11 36 03" src="https://github.com/redhat-appstudio/remote-secret/assets/1614429/5cd94bc1-83b5-428c-be4a-12e234ae3a79">
<img width="828" alt="Знімок екрана 2023-08-21 о 11 36 30" src="https://github.com/redhat-appstudio/remote-secret/assets/1614429/51ca4f26-6c90-404e-abd8-6b054a11e75e">


### What issues does this PR fix or reference?
n/a

### How to test this PR?
 - create RemoteSecret 
 - upload data. Check that status is DataObtained - true
 - delete data directly from storage.
 - Trigger RemoteSecret's reconciliation. Add annotation or restart the controller pod.
 - Check that status is DataObtained - false
